plugins/in_netif/in_netif.c

```diff
@@ -317,6 +293,31 @@ static int in_netif_init(struct flb_input_instance *in,
     return 0;
 }
 
+static struct flb_config_map config_map[] = {
+    {
+      FLB_CONFIG_MAP_STR, "interface", (char *)NULL,
+      0, FLB_TRUE, offsetof(struct flb_in_netif_config, interface),
```

Comment:
I think the data type of `interface` should be `flb_sds_t` since config_map framework allocates as `flb_sds_t`.
https://github.com/fluent/fluent-bit/blob/v1.8.12/src/flb_config_map.c#L685